dkg-runtime-primitives/src/offchain/storage_keys.rs

```diff
@@ -18,3 +18,7 @@ pub const OFFCHAIN_SIGNED_PROPOSALS: &[u8] = b"dkg-proposal-handler::signed_prop
 
 // Key for offchain storage of aggregated derived public keys
 pub const AGGREGATED_MISBEHAVIOUR_REPORTS: &[u8] = b"dkg-metadata::misbehaviour";
+
+// Key for offchain storage of aggregated derived public keys
+pub const SUBMIT_SIGNED_PROPOSAL_ON_CHAIN: &[u8] =
```

Comment:
All the lock keys shall end with the `_LOCK` postfix to allow us to know that these are used for locks.
```suggestion
pub const SUBMIT_SIGNED_PROPOSAL_ON_CHAIN_LOCK: &[u8] =
```